src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -10,22 +10,70 @@
 
 namespace Microsoft.PowerShell.Commands
 {
+    #region Restart-Computer
+
+    /// <summary>
+    /// Cmdlet to restart computer.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Restart, "Computer", SupportsShouldProcess = true,
+        HelpUri = "https://go.microsoft.com/fwlink/?LinkID=135263", RemotingCapability = RemotingCapability.SupportedByCommand)]
```

Comment:
Please use new value for `LinkID=135263`

Reply:
Done.